packages/cfd/src/Components/cfd-real-account-display.tsx

```diff
@@ -6,7 +6,8 @@ import { getAccountTypeFields, getAccountListKey, getCFDAccountKey } from '@deri
 import specifications, { TSpecifications } from '../Constants/cfd-specifications';
 import { CFDAccountCard } from './cfd-account-card';
 import { general_messages } from '../Constants/cfd-shared-strings';
-import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+// import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
```

Comment:
Pls remove commented code

Reply:
it is there because we need to update back to this type and remove `TNewDetailsOfEachMT5Loginid`